bears/java/PMDBear.py

```diff
@@ -0,0 +1,31 @@
+import re
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class PMDBear(LocalBear, Lint):
+    executable = 'run.sh'
+    arguments = "pmd -R java-basic," + \
+                "java-braces," + \
+                "java-clone," + \
+                "java-codesize," + \
+                "java-controversial," + \
+                "java-design," + \
+                "java-imports," + \
+                "java-naming," + \
+                "java-optimizations," + \
+                "java-strings," + \
+                "java-unnecessary," + \
+                "java-unusedcode" + \
+                " -d {filename}"
+    output_regex = re.compile(
+        r'(?P<file_name>.+):'
+        r'(?P<line>.+):'
+        r'(?P<message>.*)')
+
+    def run(self, filename, file):
+        '''
+        Checks java code with ``pmd``.
```

Comment:
you should mention which languages you support, really nobody knows PMD the description should give anything the user needs